README.md

```diff
@@ -1,31 +1,41 @@
-# smee &middot; [![Build Status](https://img.shields.io/travis/probot/smee/master.svg)](https://travis-ci.org/probot/smee) [![Codecov](https://img.shields.io/codecov/c/github/probot/smee.svg)](https://codecov.io/gh/probot/smee/)
+# smee-client
```

Comment:
Should we add something at the top of the README here about moving the server bits to the other repo?

Reply:
100% yep - was planning on doing that in a followup, but you're right, it should be part of this PR. Will add 👍